api.planx.uk/saveAndReturn/utils.ts

```diff
@@ -296,10 +296,10 @@ const markSessionAsSubmitted = async (sessionId: string) => {
  * Get formatted list of the session's project types
  */
 const getHumanReadableProjectType = async (
-  session: LowCalSession
+  sessionData: LowCalSession["data"]["passport"]["data"] | Record<string, any>
```

Comment:
Please see https://github.com/theopensystemslab/planx-new/pull/1633/files#r1169674642

This is a good choice 👍